README.md

```diff
@@ -96,27 +96,31 @@ to the OpenTelemetry Resource ([see below](#opentelemetry-resource)), e.g. `OTEL
 The logging exporter simply prints the name of the span along with its
 attributes to stdout. It is used mainly for testing and debugging.
 
-| System property             | Environment variable        | Purpose                                                                      |
-|-----------------------------|-----------------------------|------------------------------------------------------------------------------|
-| ota.exporter=logging        | OTA_EXPORTER=logging        | To select logging exporter                                                   |
-| ota.exporter.logging.prefix | OTA_EXPORTER_LOGGING_PREFIX | An optional string that is printed in front of the span name and attributes. |
+| System property              | Environment variable         | Purpose                                                                      |
+|------------------------------|------------------------------|------------------------------------------------------------------------------|
+| otel.exporter=logging        | OTEL_EXPORTER=logging        | To select logging exporter                                                   |
+| otel.exporter.logging.prefix | OTEL_EXPORTER_LOGGING_PREFIX | An optional string that is printed in front of the span name and attributes. |
 
 #### Propagator
 
 The propagator controls which distributed tracing header format is used.
 
-| System property | Environment variable | Purpose                                                                                 |
-|-----------------|----------------------|-----------------------------------------------------------------------------------------|
-| ota.propagators | OTA_PROPAGATORS      | Default is "tracecontext" (W3C). Other supported values are "b3", "b3single", "jaeger". |
+If this is set to a comma-delimited list of the values, the multi-propagator will be used.
+The multi-propagator will try to extract the context from incoming requests using each of the configured propagator formats (in order), stopping after the first successful context extraction.
```

Comment:
Maybe just reference the javadoc of that class?

Reply:
I thought the text description was nice, and doesn't scare away Ops folks who are not java developers 😄